GSA-TTS/federal-website-standards
# Fix header flash

## Context
On slower initial page loads the banner at the top of the page appears unfurled before the JavaScript is loaded to hide it. It's easiest to see if you emulate mid-tier mobile throttling and reload the page. Setting the accordion content to be hidden fixes this without affecting functionality.

This PR also updates a description field in `package.json`.

## How to verify this change
1. [Visit the preview page](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/deps/), open dev tools, click over to the network tab, and emulate "slow 4g". The site will take upwards of 10 seconds to load but you shouldn't ever see the gov banner expanded at any point unless you click "Here's how you know" after the page loads.
